src/System.Transactions/src/System/Transactions/SafeIUnknown.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Transactions
+{
+    // Keep an interface pointer that will not be used in a SafeHandle derived so
+    // that it will be properly released.
+    internal sealed class SafeIUnknown : SafeHandle
+    {
+        private SafeIUnknown() : base(IntPtr.Zero, true) { }
```

Comment:
Initially I made it private because nothing was using this ctor to instantiate it; the parameterless ctor was only necessary for the P/Invokes that were creating it, and for that purpose private is fine.

But then I subsequently removed all of those P/Invokes, and now nothing is using this type at all, so I'll just remove the file entirely.